src/main/java/ua/realestate/management/service/BuilderService.java

```diff
@@ -0,0 +1,48 @@
+package ua.realestate.management.service;
```

Comment:
It might be much more useful to have `db/services` package, cause it's might be unclear what kind of services you store here?

Reply:
fixed